CMakeLists.txt

```diff
@@ -82,6 +82,14 @@ if(WIN32)
   set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY_MINSIZEREL ${WINDOWS_APPLICATION_OUT_DIRECTORY})
 endif()
 
+# FreeBSD support
+
+# link BSD threading
+if(${CMAKE_SYSTEM_NAME} MATCHES "FreeBSD")
```

Comment:
I think it should be `STREQUAL` instead of `MATCHES` (the latter is for regexes).